frontend/src/types.ts

```diff
@@ -1102,6 +1103,8 @@ export interface FilterType {
     breakdown_attribution_type?: BreakdownAttributionType // funnels breakdown attribution type
     breakdown_attribution_value?: number // funnels breakdown attribution specific step value
     breakdown_histogram_bin_count?: number // trends breakdown histogram bin count
+
+    y_axis_format?: YAxisFormat
```

Comment:
The empty line seems extraneous

Reply:
It's the size of that type. Makes me want to impose structure. But that's a separate task :)

Reply:
Oh yeah, I agree about that 😄